services/mentor_service.js

```diff
@@ -10,6 +10,7 @@ const baseFields = [
   'mentors.user_id',
   'mentors.about',
   'mentors.status',
+  'mentors.tags',
```

Comment:
this won't work because `mentors` table doesn't have `tags` column. tags should be aggregated separately